app/src/main/java/com/amaze/filemanager/filesystem/HybridFileParcelable.java

```diff
@@ -147,6 +147,11 @@ public boolean isDirectory() {
     return isDirectory;
   }
 
+  @Override
+  public boolean isDirectory(Context context) {
+    return isDirectory;
```

Comment:
this may lead to issues, can you just return isDirectory only in case of SMB if you've tested it properly.